include/ck/tensor_operation/gpu/device/device_batched_gemm_reduce_xdl_cshuffle.hpp

```diff
@@ -88,6 +89,25 @@ __global__ void
                                                    c_grid_desc_mblock_mperblock_nblock_nperblock,
                                                    d_grid_desc_mblock_mperblock,
                                                    block_2_ctile_map);
+#else
+	std::ignore = p_a_grid;
```

Comment:
```ignore``` instead of ```std::ignore```, so std doesn't propagate into gpu code

```ignore``` is defined here https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/develop/include/ck/utility/ignore.hpp

FYI, there is a WIP coding style guide 
https://github.com/ROCmSoftwarePlatform/composable_kernel/wiki/Coding-Style

Reply:
Unfortunately, compiler throws errors when I tried to use just "ignore":

In file included from /root/composable_kernel/example/05_conv2d_fwd/conv2d_fwd_xdl_fp16.cpp:16:
In file included from /root/composable_kernel/include/ck/tensor_operation/gpu/device/device_conv2d_fwd_xdl_c_shuffle_nhwc_kyxc_nhwk.hpp:14:
/root/composable_kernel/include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v3r1.hpp:64:2: error: use of undeclared identifier 'ignore'; did you mean 'std::ignore'?
        ignore = a_grid_desc_ak0_m_ak1;
        ^~~~~~
        std::ignore

It works fine with std::ignore, though.